lib/rspec/core/option_parser.rb

```diff
@@ -291,6 +291,8 @@ def initialize_project_and_exit
     def bisect_and_exit(argument)
       RSpec::Support.require_rspec_core "bisect/coordinator"
 
+      # This might be the best spot to mutate ENV['SPEC_OPTS']
+      # if we do that.
```

Comment:
Only a single `Runner` instance is created for the entirety of the bisect run, so it's only going to happen once, regardless.  IMO, the logic (if we take this approach) belongs better in `Runner` than here.